lib/ae_mdw/db/mutations/name_claim_mutation.ex

```diff
@@ -120,33 +120,49 @@ defmodule AeMdw.Db.NameClaimMutation do
         auction_end = height + timeout
         m_auction_exp = Model.expiration(index: {auction_end, plain_name})
 
-        m_auction_bid =
-          Model.auction_bid(
-            index: plain_name,
-            block_index_txi_idx: {block_index, txi_idx},
-            expire_height: auction_end,
-            owner: owner_pk
-          )
-
-        auction_claim = Model.auction_bid_claim(index: {plain_name, auction_end, txi_idx})
-
         state3 =
           IntTransfer.fee(state2, {height, txi_idx}, :spend_name, owner_pk, txi_idx, name_fee)
 
         state4 =
           case State.get(state3, Model.AuctionBid, plain_name) do
             :not_found ->
-              State.inc_stat(state3, :auctions_started)
+              auction_claim = Model.auction_bid_claim(index: {plain_name, height, txi_idx})
+
+              m_auction_bid =
+                Model.auction_bid(
+                  index: plain_name,
+                  start_height: height,
+                  block_index_txi_idx: {block_index, txi_idx},
+                  expire_height: auction_end,
+                  owner: owner_pk
+                )
+
+              state3
+              |> State.inc_stat(:auctions_started)
+              |> State.put(Model.AuctionBidClaim, auction_claim)
+              |> State.put(Model.AuctionBid, m_auction_bid)
 
             {:ok,
              Model.auction_bid(
+               start_height: start_height,
                block_index_txi_idx: {_bi, prev_txi_idx},
                expire_height: prev_auction_end,
                owner: prev_owner
              )} ->
               prev_name_claim_tx = DbUtil.read_node_tx(state, prev_txi_idx)
               prev_name_fee = :aens_claim_tx.name_fee(prev_name_claim_tx)
 
+              auction_claim = Model.auction_bid_claim(index: {plain_name, start_height, txi_idx})
+
+              m_auction_bid =
+                Model.auction_bid(
+                  index: plain_name,
+                  start_height: start_height,
+                  block_index_txi_idx: {block_index, txi_idx},
+                  expire_height: auction_end,
+                  owner: owner_pk
+                )
```

Comment:
Not an operational issue but as the purpose here is an update, would use the original record and set only fields that needs to be updated to make it more clear what it is changing.

Reply:
Done